src/Foundations/GridSystem.mdx

```diff
@@ -7,47 +7,97 @@ import { Playground } from "docz";
 
 # Grid System
 
+## Breakpoints
+
+Breakpoints are the minimum and maximum widths at which a browser responds to different screen sizes, orientations and devices. The breakpoints listed below are the Bluebird Design System recommendations based on common widths of devices. As a rule for best practice, create any designs at the lowest end of the breakpoint. This will help ensure any design will not break on the lowest end of the breakpoint spectrum.
+
+![Breakpoints](../assets/Breakpoints@2x.png)
+
+| Design Token         | Display Size         | Value         |
+| -------------------- | -------------------- | ------------- |
+| `$screen-xs-min`     | Extra Small          | 480px         |
+| `$screen-sm-min`     | Small                | 768px         |
+| `$screen-md-min`     | Medium               | 992px         |
+| `$screen-lg-min`     | Large                | 1200px        |
+| `$screen-xl-desktop` | Extra Large          | 1440px        |
+
+## Media Queries
+Media queries can be used to manipulate or modify a screen. Media queries for screen widths should always be written using the `min-width` property.
+
+| Media Query                                      | What it does                          |
+| ------------------------------------------------ | ------------------------------------- |
+| `@media (min-width: $screen-xs-min) {...}`       | Targets screen widths 480px and up    |
+| `@media (min-width: $screen-sm-min) {...}`       | Targets screen widths 768px and up    |
+| `@media (min-width: $screen-md-min) {...}`       | Targets screen widths 992px and up    |
+| `@media (min-width: $screen-lg-min) {...}`       | Targets screen widths 1200px and up   |
+| `@media (min-width: $screen-xl-desktop) {...}`   | Targets screen widths 1440px and up   |
+
 ## Basic Grid
 
-Bootstrap's grid system consists of `<Row>` elements that contain `<Col>` elements. Columns can be split into segments that sum to 12, according to the screen size. So, `<Col lg={6}>` is a column that on large screens is 6/12 (or 1/2) the size of the row. `<Col md={9}>` is a column that on medium and large screens is 9/12 (or 3/4) the size of the row. By mixing and matching screen sizes you can adjust the columns to collapse in the desired fashion. Also common is to hide or show columns at certain screen sizes, eg. `xsHidden`. Rows with columns that sum to more than 12 automatically cascade.
+Bootstrap's grid system consists of `<Row>` elements that contain `<Col>` elements. 
+
+![GridRowsColumns](../assets/GridRowsColumns@2x.png)
```

Comment:
This could be demonstrated more consistently with the example below using the actual components as is done in lines 51-102.

Bonus points: improve the style of each of these grid indicators by applying better styling so they look more like the images you have generated.